spring-boot-project/spring-boot-docs/src/main/java/org/springframework/boot/docs/hazelcast/HazelcastClientConfigCustomizerConfiguration.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2012-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.docs.hazelcast;
+
+import com.hazelcast.spring.context.SpringManagedContext;
+
+import org.springframework.boot.autoconfigure.hazelcast.HazelcastClientConfigCustomizer;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * An example configuration for customizing Hazelcast Client Config.
+ *
+ * @author Dmytro Nosan
+ */
+// tag::configuration[]
+@Configuration(proxyBeanMethods = false)
+public class HazelcastClientConfigCustomizerConfiguration {
+
+	@Bean
+	public HazelcastClientConfigCustomizer hazelcastClientConfigCustomizer(ApplicationContext applicationContext) {
+		return (clientConfig) -> {
+			SpringManagedContext managedContext = new SpringManagedContext();
+			managedContext.setApplicationContext(applicationContext);
+			clientConfig.setManagedContext(managedContext);
```

Comment:
Does it make sense to enable the management context in a Hazelcast client?

Reply:
Generally speaking, I don't know.

Reply:
@jerrinot can you please help here and let us know if it does?